matrix_client/client.py

```diff
@@ -329,7 +329,7 @@ def __init__(self, client, room_id):
             NOTE: This does not verify the room with the Home Server.
         """
         if not (room_id.startswith("!") and room_id.count(":") == 1):
-            raise ValueError("RoomID's start with ! and seperate with :")
+            raise ValueError("RoomID's start with ! and separated with :")
```

Comment:
Should be along the lines of "Room IDs start with '!' and are separated with ':'".